### DISCUSSION TITLE
BUG] animations not working

### DISCUSSION THREAD

**Discussion by hoshi411** [senshu/Sozi#616] [General]
VERSION: 22.02.03-164+
Here is my project:
https://binary-personality.github.io/index.html

I have set an animation for every frame but the only animation that plays is from the phenomenal world button.
Clicking on any of the other buttons gives me no animation.

**Comment by aumouvantsillage** [comment]
If this is a bug report, please [open an issue](https://github.com/sozi-projects/Sozi/issues) and provide as much information as possible to help me understand and fix your problem.

**Reply by hoshi411** [reply]
OK.  I'll share when I can get to my desktop.
